tests/multi_asic_intfutil_test.py

```diff
@@ -0,0 +1,182 @@
+   Ethernet-BP4  97,98,99,100      40G   9100    N/A  Ethernet-BP4  PortChannel4001      up       up     N/A         off
+PortChannel1002           N/A      80G   9100    N/A           N/A           routed      up       up     N/A         N/A
+PortChannel4001           N/A      80G   9100    N/A           N/A           routed      up       up     N/A         N/A
+"""
+intf_description = """\
+  Interface    Oper    Admin        Alias               Description
+-----------  ------  -------  -----------  ------------------------
+  Ethernet0      up       up  Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+  Ethernet4      up       up  Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+"""
+
+intf_description_all = """\
+     Interface    Oper    Admin           Alias               Description
+--------------  ------  -------  --------------  ------------------------
+     Ethernet0      up       up     Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+     Ethernet4      up       up     Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+  Ethernet-BP0      up       up    Ethernet-BP0          ASIC1:Eth0-ASIC1
+  Ethernet-BP4      up       up    Ethernet-BP4          ASIC1:Eth1-ASIC1
+Ethernet-BP256      up       up  Ethernet-BP256         ASIC0:Eth16-ASIC0
+Ethernet-BP260      up       up  Ethernet-BP260         ASIC0:Eth17-ASIC0
+"""
+
+intf_description_asic0 = """\
+  Interface    Oper    Admin        Alias               Description
+-----------  ------  -------  -----------  ------------------------
+  Ethernet0      up       up  Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+  Ethernet4      up       up  Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+"""
+
+intf_description_asic0_all = """\
+   Interface    Oper    Admin         Alias               Description
+------------  ------  -------  ------------  ------------------------
+   Ethernet0      up       up   Ethernet1/1  ARISTA01T2:Ethernet3/1/1
+   Ethernet4      up       up   Ethernet1/2  ARISTA01T2:Ethernet3/2/1
+Ethernet-BP0      up       up  Ethernet-BP0          ASIC1:Eth0-ASIC1
+Ethernet-BP4      up       up  Ethernet-BP4          ASIC1:Eth1-ASIC1
+"""
+
+intf_invalid_asic_error="""ValueError: Unknown Namespace asic99"""
+
+class TestInterfacesMultiAsic(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["PATH"] += os.pathsep + scripts_path
+        os.environ["UTILITIES_UNIT_TESTING"] = "2"
+        os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+
+    def setUp(self):
+        self.runner = CliRunner()
+
+    def get_result_and_return_code(self, cmd):
+        return_code = 0
+        try:
+            output = subprocess.check_output(
+            cmd, stderr=subprocess.STDOUT, shell=True)
+        except subprocess.CalledProcessError as e: 
+            return_code = e.returncode
+            #store only the error, no need for the traceback
+            output = e.output.strip().split("\n")[-1]
+
+        return(return_code, output)
```

Comment:
This is a good candidate to be a shared function between many tests.In the future, we can maybe consider creating a base class and adding this to it.